pkg/cnb/image_rebaser.go

```diff
@@ -49,32 +48,23 @@ func (f *ImageRebaser) Rebase(build *v1alpha1.Build, ctx context.Context) (Built
 		return BuiltImage{}, errors.Wrap(err, "unsupported builder metadata structure")
 	}
 
-	newBaseImage, err := f.RemoteImageFactory.NewRemote(metadata.Stack.RunImage.Image, registry.SecretRef{
+	newBaseImage, err := f.RemoteImageFactory.newRemote(metadata.Stack.RunImage.Image, metadata.Stack.RunImage.Image, registry.SecretRef{
 		Namespace:        build.Namespace,
 		ImagePullSecrets: build.Spec.Builder.ImagePullSecrets,
-		ServiceAccount:   build.Spec.ServiceAccount,
 	})
 	if err != nil {
-		return BuiltImage{}, errors.Wrap(err, "unable to fetch remote run image")
+		return BuiltImage{}, err
```

Comment:
Why did we stop wrapping this error?

Reply:
`newRemote` wraps the error. No need to wrap it again.